### ISSUE TITLE
Affects of smaller batch size when using Batch Normalizations.

### ISSUE THREAD

**Issue by teamteam321** [google/automl#881]
I have training this model for several months on a GPU with small memory (8 GB 1070ti), on D2 my configuration couldn't goes beyone batch size of 3. 

The results were good but I noticed that on D0 (batch size of 12)and D1 (batch size of 6) the recall of both are better then d2 even though both of their AP and AP50 are significantly lower ,  And from this paper https://arxiv.org/pdf/1803.08494.pdf, Does smaller batch size are not good with batch normalization? and if it's happened to be the case. Is it possible to implement Group normalization?

I am by no means an expert and happy to listen if i'm missing something.

.

Here are some result from tensorboard with identical datasets and params training on 20k train images and 5k eval images (average >1 bbox / images). both d0 with 20 epochs, d1 with 15 epochs and d2 with 7 epochs

d0, d0 (@ batch size of 3), d1 and d2

![recall_max1](https://user-images.githubusercontent.com/36959658/99878276-cdf25900-2c36-11eb-9aea-827ad2789dbe.PNG)

![ap50](https://user-images.githubusercontent.com/36959658/99878286-d77bc100-2c36-11eb-9a8a-d569d3a4f09c.PNG)

![box_loss](https://user-images.githubusercontent.com/36959658/99878297-ef534500-2c36-11eb-9a95-6e2c1fc1ac82.PNG)

![class_loss](https://user-images.githubusercontent.com/36959658/99878300-f1b59f00-2c36-11eb-807a-77fa0f0941a0.PNG)

...Sorry for my bad English.
